packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -667,6 +668,23 @@ export interface LoadContainerRuntimeParams {
 	 * @deprecated Will be removed once Loader LTS version is "2.0.0-internal.7.0.0". Migrate all usage of IFluidRouter to the "entryPoint" pattern. Refer to Removing-IFluidRouter.md
 	 * */
 	requestHandler?: (request: IRequest, runtime: IContainerRuntime) => Promise<IResponse>;
+
+	/**
+	 * minVersionForCollab is used to determine the default configuration for {@link IContainerRuntimeOptionsInternal}
+	 * properties which affect the document schema. minVersionForCollab can be considered to be the minimum version of
+	 * the FF runtime that we should support compatibility with.
```

Comment:
Nit (I generally recommend against using the name of an API item in its own docs - doing so is redundant and is fragile to API renames):
```suggestion
	 * Used to determine the default configuration for {@link IContainerRuntimeOptionsInternal}
	 * properties, which affect the document schema. It can be considered to be the minimum version of
	 * the FF runtime that we should support compatibility with.
```